lib/oracle.js

```diff
@@ -150,61 +179,93 @@ Oracle.prototype.executeSQL = function (sql, params, options, callback) {
   if (transaction && transaction.connection &&
     transaction.connector === this) {
     debug('Execute SQL within a transaction');
-    transaction.connection.execute(sql, params, function (err, data) {
-      if (err && self.settings.debug) {
-        self.debug(err);
-      }
-      if (self.settings.debug && data) {
-        self.debug("Result: %j", data);
-      }
-      callback(err, data);
-    });
+    transaction.connection.execute(sql, params,
+      {outFormat: oracle.OBJECT, autoCommit: false}, function(err, data) {
+        if (err && self.settings.debug) {
+          self.debug(err);
```

Comment:
Shall we add a name for debug message here, like `self.debug('Transaction error: ' + err)`?